hw/top_earlgrey/lint/top_earlgrey.waiver

```diff
@@ -15,6 +15,16 @@ waive -rules RESET_MUX -location {top_earlgrey.sv} -regexp {Asynchronous reset .
 waive -rules CLOCK_MUX -location {clkmgr.sv top_earlgrey.sv} -regexp {.*clk_io_div.* is driven by a multiplexer here} \
       -comment "Divided clocks go through prim_clock_div, which use muxes for scan bypass and clock step down"
 
+waive -rules {HIER_NET_NOT_READ NOT_READ} -location {clkmgr_reg_top.sv pwrmgr_reg_top.sv rstmgr_reg_top.sv} -regexp {.*(reg_wdata|reg_be).* is not read from} \
+      -comment "Depending on top level assignments, the write path may not always be fully populated or used"
```

Comment:
same question: can we fix this with a `assign unused_sigs = ^{signals, ...};` as well?